deploy/helm-chart/README.md

```diff
@@ -32,27 +32,34 @@ This chart deploys the NGINX Gateway Fabric in your Kubernetes cluster.
 
 > **Note**
 >
-> The Gateway API resources from the standard channel must be installed
-> before deploying NGINX Gateway Fabric. If they are already installed in your cluster, please ensure they are
-> the correct version as supported by the NGINX Gateway Fabric -
+> The [Gateway API resources](https://github.com/kubernetes-sigs/gateway-api) from the standard channel must be
+> installed before deploying NGINX Gateway Fabric. If they are already installed in your cluster, please ensure
+> they are the correct version as supported by the NGINX Gateway Fabric -
 > [see the Technical Specifications](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/README.md#technical-specifications).
 
-To install the Gateway API CRDs from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+If installing the latest stable release of NGINX Gateway Fabric, ensure you are deploying v0.8.1 of the
+Gateway API resources:
 
-```shell
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
-```
+  ```shell
+  kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.8.1/standard-install.yaml
+  ```
 
-If you are running on Kubernetes 1.23 or 1.24, you also need to install the validating webhook. To do so, run:
+If installing the edge version of NGINX Gateway Fabric:
 
-```shell
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/webhook-install.yaml
-```
+  ```shell
+  kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
+  ```
+
+  If you are running on Kubernetes 1.23 or 1.24, you also need to install the validating webhook. To do so, run:
+
+  ```shell
+  kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/webhook-install.yaml
+  ```
 
 > **Important**
 >
 > The validating webhook is not needed if you are running Kubernetes 1.25+. Validation is done using CEL on the
-> CRDs. See the [resource validation doc](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/resource-validation.md)
```

Comment:
Should this remain a Github link since this file lives in the README?

Reply:
The resource validation doc will live in docs.nginx.com, not github